docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -1,214 +1,215 @@
+
+. Click the *Settings* tab, make changes, and click *Save changes*
++
+Alternatively, click *Delete policy* to delete the policy.
+Existing data will not be deleted.
+Any agents assigned to a policy must be unenrolled before a policy can be deleted.
 
 [discrete]
 [[integration-updates]]
-== How to know a new Integration version exists
-
-New Integration updates are released by Elastic periodically, every month or two, generally (or as
-otherwise syncs up to a minor release version).  When the Integration 'packages' are released (the
-packages are how the Integrations are delivered to the stack, via the on-line Elastic Package Registry
-and the {kib} side Elastic Package Manager component). When a new Integration is released by Elastic
-it will show up in the Integrations listing in {kib} after a restart (like during a stack upgrade) or
-the user can trigger the refresh with the Fleet API, discussed in separate documentation.  In order
-to take advantage of a newer Integration, there are a few steps a user must take as outlined below.
-It is also recommended in larger deployments to try a new Integration with one sample Agent instead
-of all that may apply.
+== Integration updates
 
-[discrete]
-[[integration-update-steps]]
-== How to update an Integration version in a {policy}
-
-To update an Integration in a {policy} a user can follow the following steps:
-First, a user can confirm the new Integration version desired is shown in the Integrations list in {kib}
-as documented above.  If a newer Integration exists, the user can move forward with the following steps.
-Step two, a user must update the Integration in the {fleet} app.  This is an explicit action for now, as
-it can cause different or unexpected / undesired behavior on the hosts, so care is taken to make it
-explicit.  Once desired though, for step three, a user can create a new {policy} with the desired
-Integration (this will use the newer available version) and for step four a user can apply it to the {agent},
-as described in the {policy} management documentation above.  Step five consists of rolling out the change
-to any further hosts desired, after the small trial is applied and deemed acceptable.
+Elastic releases integration updates periodically.
+Integrations are delivered to the Elastic Stack, via the online Elastic Package Registry
+and the {kib} Elastic Package Manager. When a new Integration is released by Elastic,
+it will show up in the Integrations listing in {kib} after a restart (like during a stack upgrade).
+// to do: link to the Fleet API docs
+A refresh can also be triggered manually, with the Fleet API.
 
-[discrete]
-[[integration-update-steps1]]
-== How to update an Integration version in a {policy}
-
-To check if a new Integration exists, follow the {policy} management documentation steps above to view
-a {policy} with the Integration whose version is to be compared against the available Integrations
```

Comment:
lol, this section is really rambly.  thanks for just taking it out.  i will have to look to see what we have in 7.13 and what is coming in terms of 'indicators' to help users know they need to update an integration.  it is an area we are improving, so the exact state is tbd until I go look at code and prs coming.